test/raise_on_unpermitted_params_test.rb

```diff
@@ -30,4 +30,15 @@ def teardown
       params.permit(:book => [:pages])
     end
   end
-end
\ No newline at end of file
+  
+  test "not raise on params included in NEVER_UNPERMITTED_PARAMS" do
+    key = ActionController::Parameters::NEVER_UNPERMITTED_PARAMS[0]
+    params = ActionController::Parameters.new({
+      key => "Turnips"
```

Comment:
Possibly, but if the params that need testing were to change in the future then the test wouldn't pick it up either way so the test wouldn't be of much use.